packages/slice-machine/pages/_app.tsx

```diff
@@ -103,7 +106,7 @@ function MyApp({ Component, pageProps }: AppContext & AppInitialProps) {
               <LoadingPage />
             ) : (
               <Provider store={smStore.store}>
-                <ConnectedRouter Router={Router}>
+                <ConnectedRouter>
```

Comment:
Keep the router in there, if not it won't be synced with the store (that's why it was added in the first place)

Reply:
I know it's optional, look at the history and you will see we assumed the same thing 
https://github.com/prismicio/slice-machine/blame/37a455d849c551c1e0d3e752720896d7b7492cbb/packages/slice-machine/pages/_app.tsx

But or some reason the store was not syncing with the router, adding the router solved this issue.

Reply:
yey